packages/config/config/devices/0x024f/fwc01.json

```diff
@@ -0,0 +1,227 @@
+			"maxValue": 32767,
+			"defaultValue": 0
+		},
+		{
+			"#": "5",
+			"label": "Safety OFF Period",
+			"unit": "seconds",
+			"valueSize": 2,
+			"minValue": 0,
+			"maxValue": 32767,
+			"defaultValue": 600
+		},
+		{
+			"#": "6",
+			"label": "Safety ON Period",
+			"unit": "seconds",
+			"valueSize": 2,
+			"minValue": 0,
+			"maxValue": 32767,
+			"defaultValue": 300
+		},
+		{
+			"#": "10",
+			"label": "Dimming speed",
+			"description": "Dimmer Speed in 1/100th of a Second",
+			"unit": "/100s",
+			"valueSize": 2,
+			"minValue": 30,
+			"maxValue": 32767,
+			"defaultValue": 300
+		},
+		{
+			"#": "11",
+			"label": "Dimmer mode",
+			"description": "Dimmer Operation Mode",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 1,
+			"defaultValue": 1
+		},
+		{
+			"#": "12",
+			"label": "Maximum Level",
+			"valueSize": 1,
+			"minValue": 1,
+			"maxValue": 99,
+			"defaultValue": 90
+		},
+		{
+			"#": "13",
+			"label": "Minimum Level",
+			"valueSize": 1,
+			"minValue": 1,
+			"maxValue": 99,
+			"defaultValue": 15
+		},
+		{
+			"#": "14",
+			"label": "UI modes and Zones",
+			"description": "Enable UI Modes and Zones via Bits",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 67,
+			"defaultValue": 0
+		},
+		{
+			"#": "15",
+			"label": "Light Zone Control Mode",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 30,
+			"defaultValue": 0
+		},
+		{
+			"#": "17",
+			"label": "Back Unit Role",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 2,
+			"defaultValue": 1
+		},
+		{
+			"#": "18",
+			"label": "Back Unit Light Zone",
+			"valueSize": 1,
+			"minValue": 1,
+			"maxValue": 4,
+			"defaultValue": 1
+		},
+		{
+			"#": "32",
+			"label": "Thermostat Set Point Max",
+			"description": "Set Point Max in 1/10th of a degree Celcius",
+			"unit": "1/10 degrees",
+			"valueSize": 2,
+			"minValue": 0,
+			"maxValue": 500,
+			"defaultValue": 270
+		},
+		{
+			"#": "34",
+			"label": "Thermostat Set Point Min",
+			"description": "Set Point Min in 1/10th of a degree Celcius",
+			"valueSize": 2,
+			"minValue": 0,
+			"maxValue": 500,
+			"defaultValue": 50
+		},
+		{
+			"#": "36",
+			"label": "Thermostat Control Resend Interval",
+			"description": "Resend Interval in seconds",
+			"unit": "seconds",
+			"valueSize": 2,
+			"minValue": 0,
+			"maxValue": 32767,
+			"defaultValue": 3600
+		},
+		{
+			"#": "37",
+			"label": "Thermostat Minimum On/Off Interval",
+			"description": "Minimum On/Off interval in seconds",
+			"unit": "seconds",
+			"valueSize": 2,
+			"minValue": 60,
+			"maxValue": 32767,
+			"defaultValue": 300
+		},
+		{
+			"#": "38",
+			"label": "Thermostat Floor Temp Max",
+			"description": "Floor Temp Max in 1/10th of a degree Celcius",
+			"unit": "1/10 degrees",
+			"valueSize": 2,
+			"minValue": 0,
+			"maxValue": 500,
+			"defaultValue": 270
+		},
+		{
+			"#": "40",
+			"label": "Temperature Sensor Select",
+			"valueSize": 1,
+			"minValue": 1,
+			"maxValue": 3,
```

Comment:
What do these values mean? There should probably be options describing them.

Reply:
I'll add options.